### ISSUE TITLE
feat: improve a11y for remove an image from editor

### ISSUE THREAD

**Issue by ayo-run** [elk-zone/elk#248]
When writing a Toot and the user attached an image, he should be able to remove it again.

An X button to clear the attachment is usually how this is implemented.

<img width="746" alt="Screenshot 2022-11-30 at 9 39 06 AM" src="https://user-images.githubusercontent.com/4262489/204748729-6a215b2f-4544-4c2d-b765-ff8928b95e58.png">
